src/Raven.Client/Documents/Session/SessionDocumentTimeSeries.cs

```diff
@@ -30,9 +30,9 @@ public void Append(string timeseries, DateTime timestamp, string tag, IEnumerabl
             _asyncSessionTimeSeries.Append(timeseries, timestamp, tag, values);
         }
 
-        public IEnumerable<TimeSeriesEntry> Get(string timeseries, DateTime from, DateTime to)
+        public IEnumerable<TimeSeriesEntry> Get(string timeseries, DateTime from, DateTime to, int? skip = null, int? take = null)
```

Comment:
Should not be nullable. Just do `skip = 0` and `take = int.MaxValue`
1. Do we want to change that to `long` ?
2. In client API we are using `int start, int pageSize` more often